src/modules/pages/LoginPage/LoginForm.js

```diff
@@ -21,7 +24,10 @@ const propTypes = {
   isFetching: PropTypes.bool.isRequired
 }
 
-const LoginForm = ({ authorizeThroughGithub, errorMsg, isFetching, user }) => {
+const LoginForm = ({ authorizeBasic, authorizeThroughGithub, errorMsg, isFetching, user, handleSubmit }) => {
+  const DISABLE_BASIC_AUTH_FORM = (window.MAIN_CONFIG.ui.disable_basic_auth === 'true')
+  const DISABLE_GITHUB_AUTH = (window.MAIN_CONFIG.ui.disable_github_auth === 'true')
```

Comment:
`window.MAIN_CONFIG.ui.disable_github_auth` expected to be string `'true'`? Because:
```js
true === 'true' // false
```

Reply:
Yes, environment variables are strings.